content/en/docs/ops/ambient/getting-started/test.sh

```diff
@@ -65,9 +65,12 @@ _verify_contains snip_verify_traffic_notsleep_to_productpage "$snip_verify_traff
 snip_adding_your_application_to_the_ambient_mesh_1
 
 # test traffic after ambient mode is enabled
-_verify_contains snip_adding_your_application_to_the_ambient_mesh_2 "$snip_adding_your_application_to_the_ambient_mesh_2_out"
-_verify_contains snip_adding_your_application_to_the_ambient_mesh_3 "$snip_adding_your_application_to_the_ambient_mesh_3_out"
+_verify_like snip_adding_your_application_to_the_ambient_mesh_1 "$snip_adding_your_application_to_the_ambient_mesh_1_out"
+_verify_contains snip_adding_your_application_to_the_ambient_mesh_2 ""
+_verify_like snip_adding_your_application_to_the_ambient_mesh_3 "received netns, starting proxy"
```

Comment:
It looks like when you had contains you had `$snip_adding_your_application_to_the_ambient_mesh_3_out` on the line so it was checking for `... received` and there is no ... in the output, so it makes sense that it failed. IF you updated it to not be checking for the `out` string and use this string, you should be ok
```suggestion
_verify_contains snip_adding_your_application_to_the_ambient_mesh_3 "received netns, starting proxy"
```